### ISSUE TITLE
[Enhancement] "Merge" xct viewmodel to vs' tool

### ISSUE THREAD

**Issue by jingliancui** [xamarin/XamarinCommunityToolkit#799]
Description in [preview release note](https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-preview#16.9.0.pre.3.0), 
`Improved MVVM Support: In this release, we introduced a new dialog that allows you to quickly create a view-model and declare it as the DataContext for the XAML page. Setting the DataContext will enhance IntelliSense when working with data bindings – such as the ability to create properties in the view-model, Go To Definition, and completions.`
I think it's possible to add xct viewmodel to this feature.

**Comment by pictos** [comment]
@jingliancui I didn't follow your idea, can you rephrase that?

**Comment by jingliancui** [comment]
@pictos ,The vs' preview MVVM support feature like the "prop tab tab", the viewmodel it create by the operation maybe is created from vs , and I know xct support DelegateWeakEventManager, so is it possible to "embedded" the DelegateWeakEventManager feature to the vs' mvvm feature?

**Comment by pictos** [comment]
I see now, well I believe that we can't do that on this project, that should be something done by VS team. @jfversluis @jsuarezruiz thoughts?

**Comment by jingliancui** [comment]
Maybe it like the vs `code snippet` feature , or the aspnetmvc controller create dialog etc...

**Comment by jingliancui** [comment]
And I suddenly remember the` .net 5 code generator` feature.

**Comment by jfversluis** [comment]
Yeah I think this is more of a tooling thing than anything we can do :)

If I understand your idea correctly a number of things need to happen. First, the Toolkit needs to become a part of the templates to be installed whenever someone creates a new project and then they need to add the logic to generate this code.

I highly doubt that we will ever get that far with this project, but I appreciate your big plans! For now I don't see how this will happen anytime soon and it's definitely not something we can do ourselves. Closing this for now.
